mountpoint-s3/src/inode.rs

```diff
@@ -487,7 +497,10 @@ impl SuperblockInner {
                 debug_assert!(false, "inodes never change kind");
                 return Err(InodeError::NotADirectory(parent.ino()));
             }
-            InodeKindData::Directory { children } => {
+            InodeKindData::Directory {
+                children,
+                writing_children: _,
+            } => {
```

Comment:
Here and everywhere else:
```suggestion
            InodeKindData::Directory { children, .. } => {
```